shell/platform/android/io/flutter/embedding/engine/FlutterEngine.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.embedding.engine;
+
+/**
+ * WARNING: THIS CLASS IS EXPERIMENTAL. DO NOT SHIP A DEPENDENCY ON THIS CODE.
+ * IF YOU USE IT, WE WILL BREAK YOU.
+ */
+public class FlutterEngine {
```

Comment:
nit: since we're introducing the class as part of this PR how about a very brief description of what is the `FlutterEngine` (going to be) in the comment.

Reply:
I migrated the JavaDoc from the staging branch into this one, so now there is a JavaDoc for FlutterEngine.